debugger/debugger.js

```diff
@@ -0,0 +1,310 @@
+/**
+ * Debugger support for skulpt module
+ */
+
+var Sk = Sk || {}; //jshint ignore:line
+
+function hasOwnProperty(obj, prop) {
+    var proto = obj.__proto__ || obj.constructor.prototype;
```

Comment:
this `__proto__` property is something google closure adds? I don't remember ever running into it in skulpt.